addons/l10n_it_edi/views/report_invoice.xml

```diff
@@ -2,12 +2,19 @@
 <odoo>
     <template id="report_invoice_document" inherit_id="account.report_invoice_document">
         <div name="comment" position="before">
-            <div t-if="o.l10n_it_origin_document_type" name="pa_fields">
-                <b><span t-field="o.l10n_it_origin_document_type"/></b>: <span t-field="o.l10n_it_origin_document_name"/><br/>
-                <t t-if="o.l10n_it_origin_document_date"><b>Document Date: </b><span t-field="o.l10n_it_origin_document_date"/><br/></t>
-                <t t-if="o.l10n_it_cig"><b>CIG: </b><span t-field="o.l10n_it_cig"/><br/></t>
-                <t t-if="o.l10n_it_cup"><b>CUP: </b><span t-field="o.l10n_it_cup"/><br/></t>
-            </div>
+                <t t-if="o.l10n_it_stamp_duty"><b>Stamp Duty: </b><span t-field="o.l10n_it_stamp_duty"></span></t><br/>
```

Comment:
`<br/>` is not included in the t-if condition. Are you not addind extra empty line in case of missing data?